ocean_lib/config.py

```diff
@@ -205,27 +195,13 @@ def _load_environ(self):
                 self._logger.debug(f"Config: setting environ {option_name} = {value}")
                 self.set(environ_item[2], option_name, value)
 
-    @property
-    def artifacts_path(self):
-        """Path where the contracts artifacts are allocated."""
-        _path_string = self.get(SECTION_ETH_NETWORK, NAME_ARTIFACTS_PATH)
-        path = Path(_path_string).expanduser().resolve() if _path_string else None
-
-        if path and path.exists():
-            return path
-
-        return Path(artifacts.__file__).parent.expanduser().resolve()
-
     @property
     def address_file(self):
```

Comment:
I think `config.address_file` should fall back to the `address.json` included with `ocean-contracts` if not specified via envvar or config option, like so:

```python
import artifacts
...snip...
    @property
    def address_file(self):
        file_path = self.get(SECTION_ETH_NETWORK, NAME_ADDRESS_FILE)
        if file_path:
            file_path = Path(file_path).expanduser().resolve()
        else:
            file_path = Path(artifacts.__file__).parent.expanduser().resolve()
        return str(file_path)
```

This logic should work because `DEFAULT_ADDRESS_FILE = ""` (empty string) which evaluates to `False`.